src/components/cards/EthCard.tsx

```diff
@@ -57,27 +57,24 @@ export const EthCard = () => {
     emitChartsRequest([ETH_ADDRESS], chartTypes.day, nativeCurrency);
   }, [nativeCurrency]);
 
-  const handlePressBuy = useCallback(() => {
-    if (isDamaged) {
-      showWalletErrorAlert();
-      return;
-    }
+  const handlePressBuy = useCallback(
+    e => {
+      e.stopPropagation();
```

Comment:
Required to prevent a click on the button from firing the click on the outer card itself.